src/TimeSteppers/velocity_and_tracer_tendencies.jl

```diff
@@ -0,0 +1,130 @@
+"""
+    u_velocity_tendency(i, j, k, grid, coriolis, surface_waves, 
+                        closure, U, C, K, F, pHY′, parameters, time)
+
+Return the tendency for the horizontal velocity in the x-direction, or the east-west 
+direction, ``u``, at grid point `i, j, k`.
+
+The tendency for ``u`` is called ``G_u`` and defined via
+
+    ``∂_t u = G_u - ∂_x ϕ_n``
+
+where ∂_x ϕ_n is the non-hydrostatic pressure gradient in the x-direction.
+
+`coriolis`, `surface_waves`, and `closure` are types encoding information about Coriolis
+forces, surface waves, and the prescribed turbulence closure.
+
+The arguments `U`, `C`, and `K` are `NamedTuple`s with the three velocity components,
+tracer fields, and precalculated diffusivities where applicable. `F` is a named tuple of 
+forcing functions, `pHY′` is the hydrostatic pressure anomaly.
+
+`parameters` is a `NamedTuple` of scalar parameters for user-defined forcing functions 
+and `time` is the physical time of the model.
```

Comment:
I thought `model.parameters` could be anything passed in by the user, although we usually use named tuples (so do most people for these kinds of parameters).

Reply:
That's true about `model.parameters` --- except that kernels will fail to compile on the GPU unless it only contains simple objects.

Reply:
Hmm, ok, I will change the docstring to be more accurate.

Reply:
I think we're planning to get rid of `model.parameters` but whatever we replace it with should probably work the same way for writing forcing functions so thankfully the docstrings will still be correct then.

Reply:
Should we merge this and address when we address #682 ?